src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -56,9 +56,6 @@ enum DSSurveyLabelIndex {
     No
 }
 
-const NotebookOpenThreshold = 5;
-const NotebookExecutionThreshold = 100;
```

Comment:
Why are sweet removing this?

Reply:
They're on `shouldShowBanner` they now depend on the type of survey

Reply:
here's why:
https://github.com/microsoft/vscode-jupyter/issues/5261#issuecomment-824438141